### PR TITLE
Update Presto/Hive MeteringConfig schema

### PR DESCRIPTION
**Pull request by chancez**
- Moves Hive into it's own top-level configuration section
- Un-nests Presto's configuration now that it only contains Presto options
- Updates Hive's configuration schema
- Updates Presto's configuration schema
- Updates Hadoop's AWS configuration schema
- AWS configuration is now consistent across all configuration sections

### CODE REVIEW

**Comment by chancez** [comment]
/hold
Going to wait to see if other PRs should be merged first.

File: images/metering-ansible-operator/roles/meteringconfig/tasks/deploy_resources.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     loop_var: resource
     label: "{{ resource.template_file }}"
   when: resource.create | default(true)
-  no_log: true
+  # no_log: true
```
Comment (timflannagan1) [inline images/metering-ansible-operator/roles/meteringconfig/tasks/deploy_resources.yml]: this is commented out

Comment (chancez) [reply images/metering-ansible-operator/roles/meteringconfig/tasks/deploy_resources.yml]: Dangit

File: charts/openshift-metering/templates/hive/hive-configmap.yaml [hunk 2]
```diff
@@ -31,46 +31,52 @@ data:
       </property>
       <property>
         <name>hive.metastore.uris</name>
-        <value>{{ .Values.presto.spec.hive.config.metastoreURIs }}</value>
+        <value>{{ .Values.hive.spec.config.metastoreURIs }}</value>
       </property>
```
Comment (EmilyM1) [inline charts/openshift-metering/templates/hive/hive-configmap.yaml]: I don't see in hive.spec.config the metastoreURIS field

Comment (EmilyM1) [reply charts/openshift-metering/templates/hive/hive-configmap.yaml]: oh nvm... see it!

Comment (chancez) [reply charts/openshift-metering/templates/hive/hive-configmap.yaml]: https://github.com/operator-framework/operator-metering/pull/783/files#diff-60c0d87b6c8a5d942ed6b466067f8dfdR574

File: images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml [hunk 3]
```diff
@@ -58,25 +60,30 @@ _storage_overrides:
...
           # create buckets by default
           createBucket: true
           region: ""
     reporting-operator:
       spec:
         config:
           aws:
-            secretName: "{{ _storage_spec | json_query('hive.s3.awsCredentialsSecretName') }}"
+            secretName: "{{ _storage_spec | json_query('hive.s3.secretName') }}"
             createSecret: false
```
Comment (EmilyM1) [inline images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml]: Is _storage_spec or hive.s3 in the values.yaml?

Comment (chancez) [reply images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml]: `_storage_spec` is a variable in ansible defined above that contains `spec.storage`. This is querying the `spec.storage` section and getting `spec.storage.hive.s3.secretName`

Comment (EmilyM1) [reply images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml]: 👍

**Pull request merged**